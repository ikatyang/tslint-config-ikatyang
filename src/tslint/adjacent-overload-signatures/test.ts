// good

interface I {
  a();
  a();
  b();
}

// bad

interface I {
  a();
  b();
  a();
}
