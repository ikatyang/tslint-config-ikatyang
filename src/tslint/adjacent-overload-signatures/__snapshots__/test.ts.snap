// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

interface I {
  a();
  a();
  b();
}

// bad

interface I {
  a();
  b();
  a();
  ~~~~ [All 'a' signatures should be adjacent]
}

`;
